controller/Controller.class.php

```diff
@@ -198,4 +198,29 @@ public static function shutdown()
             call_user_func($fn);
         }
     }
+
+    protected static function doLowerUriRedirectOr404($uri)
+    {
+        $router = static::getRouterWithRoutes();
+        // $lowerUri = strtolower($uri);
+        $lowerUri = '/news/lbry-evolves';
```

Comment:
Why does this function specify this URL?

Reply:
I hard coded it during one of the test.

Reply:
Tests shouldn't be in the final code.